webrtc.html

```diff
@@ -1450,7 +1493,8 @@
                       </li>
                       <li>
                         <p>In parallel, start the process to generate an SDP
-                        answer, as described in <span data-jsep=
+                        answer that reflects changes in <var>answerBundle</var>,
+                        as described in <span data-jsep=
                         "generating-an-answer">[[!JSEP]]</span>.</p>
```

Comment:
What happens with this?

setRemoteDescription(offer);
createAnswer();

Does that work?  

setRemoteDescription(offer);
createAnswer();
addTrack();

Does it affect setRemoteDescription?  Does it affect createAnswer?


I think this isn't going to work.